packages/fetch-error-handler/test/unit/lib/create-handler.spec.js

```diff
@@ -168,6 +173,34 @@ describe('@dotcom-reliability-kit/fetch-error-handler', () => {
 						expect(error).toBeInstanceOf(Error);
 					}
 				});
+
+				describe('when the response body has a pipe method (node-fetch)', () => {
+					it('pipes the body into a stream that voids the body data', async () => {
+						expect.assertions(5);
+						try {
+							mockResponse.ok = false;
+							mockResponse.status = 400;
+							mockResponse.body.pipe = jest.fn();
+							await fetchErrorHandler(Promise.resolve(mockResponse));
+						} catch (error) {
+							expect(mockResponse.body.pipe).toHaveBeenCalledTimes(1);
+							expect(mockResponse.body.pipe).toHaveBeenCalledWith(
+								expect.any(Writable)
+							);
+							const stream = mockResponse.body.pipe.mock.calls[0][0];
+							expect(typeof stream._write).toEqual('function');
+
+							// We make sure the _write function doesn't do anything with
+							// any of the data aside from calling `done`
+							const chunk = {}; // The test will error if we try to call any chunk methods, e.g. toString
+							const encoding = {}; // The test will error if we try to perform string operations on the encoding
+							const done = jest.fn();
+							stream._write(chunk, encoding, done);
+							expect(done).toHaveBeenCalledTimes(1);
+							expect(done).toHaveBeenCalledWith();
+						}
```

Comment:
Is the `.toHaveBeenCalledWith()` meant to be empty?

Reply:
Yep 👍  because we expect `done` to be called with no arguments. Unfortunately there's not a nice Jesty way to do this where it's clearer what we're doing. An alternative would be this but I think it's possible less clear, thoughts?

```js
expect(done.mock.calls[0].length).toEqual(0);
```

Alternatively I could add a comment to clarify if you like?

Reply:
Ah I see, ok this makes sense. Now explained, the original code is fine. I think a comment would suffice 😌